library/src/main/java/io/ona/kujaku/data/realm/RealmDatabase.java

```diff
@@ -33,4 +35,33 @@ private RealmDatabase(Context context) {
                 .build();
         Realm.setDefaultConfiguration(configuration);
     }
+
+    public boolean deleteTask(@NonNull String mapName, boolean isDownloadTask) {
+        Realm realm = Realm.getDefaultInstance();
+        MapBoxOfflineQueueTask taskToDelete;
+        if (isDownloadTask) {
```

Comment:
how about removing the repetition by only setting the `taskType` in the conditional:

```java
MapBoxOfflineQueueTask taskType = isDownloadTask ? MapBoxOfflineQueueTask.TASK_TYPE_DOWNLOAD : MapBoxOfflineQueueTask.TASK_TYPE_DELETE

taskToDelete = realm.where(MapBoxOfflineQueueTask.class)
                    .equalTo("taskType", taskType)
                    .contains("task", mapName)
                    .findFirst();
```

Reply:
Btw, yea